src/lwc/geFormRenderer/geFormRenderer.js

```diff
@@ -66,6 +77,7 @@ export default class GeFormRenderer extends NavigationMixin(LightningElement) {
     @track mappingSet = '';
     @track version = '';
     @track formTemplateId;
+    @track batchDefaults;
```

Comment:
If this field doesn't need to be explicitly reactive, it could be an underscored "private" var `_batchDefaults` (even though all fields will be reactive from Spring '20 forward...)

Reply:
Thanks Ken, you've got a point. We're now fully on Spring '20 and the it's not meant to be reactive. I just updated it.